packages/valory/skills/transaction_settlement_abci/tests/test_payload_tools.py

```diff
@@ -101,105 +92,3 @@ def test_payload_to_hex_and_back() -> None:
 
     intermediate = hash_payload_to_hex(**tx_params)  # type: ignore
     assert tx_params == skill_input_hex_to_payload(intermediate)
-
-
-@pytest.mark.skip
-def test_fuzz_tx_hist_payload_to_hex() -> None:
-    """Test fuzz tx_hist_payload_to_hex."""
-
-    @atheris.instrument_func
-    def fuzz_tx_hist_payload_to_hex(input_bytes: bytes) -> None:
-        """Fuzz tx_hist_payload_to_hex."""
-        fdp = atheris.FuzzedDataProvider(input_bytes)
-        verification_int = fdp.ConsumeInt(4)
-        if verification_int < 1 or verification_int > 6:
-            return
-        verification = VerificationStatus(verification_int)
-        tx_hash = fdp.ConsumeString(12)
-        if len(tx_hash) != 64:
-            return
-        tx_hist_payload_to_hex(verification, tx_hash)
-
-    atheris.instrument_all()
-    atheris.Setup(sys.argv, fuzz_tx_hist_payload_to_hex)
-    atheris.Fuzz()
```

Comment:
This piece of code does not warrant the use of a fuzzer
https://github.com/valory-xyz/open-autonomy/blob/faabe6263607775720aad863211d0ca6fd17d937/packages/valory/skills/transaction_settlement_abci/payload_tools.py#L58-L70